dbinit/src/main/liquibase/updates/changelog-0004-yd-417.yml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+- changeSet:
+    id: 1490127260476-1
+    author: avdijk (generated)
+    changes:
+    - addColumn:
+        columns:
+        - column:
+            name: last_status_change_time
+            type: varchar(255)
+        tableName: buddies
+- changeSet:
+    id: 1490127260476-21
+    author: avdijk (generated)
+    changes:
+    - dropColumn:
```

Comment:
This is not going to work. It would cause all users to loose their "last status change time" for their buddies.
We need to retain both columns till we have migrated the data for all users from their ``BuddyAnonymized`` entity to their ``Buddy`` entity.